blaze-core/src/main/scala/org/http4s/blaze/util/BodylessWriter.scala

```diff
@@ -1,4 +1,6 @@
-package org.http4s.blaze.util
```

Comment:
Any particular reason for this change? I don't think I had a consistent style, but I don't see a reason associated with this one.